string env_name
string plugin_name
---
bool success
string error
